### PR TITLE
Citizen decisions list E2E test

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

**Review by jaahon — COMMENTED** [review]
The citizen tests don't clean up after themselves properly, you can't run the tests twice.

**Comment by reynders** [comment]
> The citizen tests don't clean up after themselves properly, you can't run the tests twice.

I believe this is the case with all of our e2e tests? I could add the cleanup to this PR (basically a full db reset), but it is unrelated with the decision functionality. Or did you have a specific case in mind?

File: frontend/e2e-test/test/e2e/specs/0_citizen/citizen-decisions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const citizenHomePage = new CitizenHomePage()
+const citizenDecisionsPage = new CitizenDecisionsPage()
+
+let applicationId: string
+let employee: EmployeeBuilder
+
+fixture('Citizen decisions')
+  .meta({ type: 'regression', subType: 'citizen-decisions' })
+  .before(async () => {
+    await initializeAreaAndPersonData()
```
Comment (jaahon) [inline frontend/e2e-test/test/e2e/specs/0_citizen/citizen-decisions.spec.ts:29]: This data is not cleaned up afterwards. The function returns a function that does it.

Comment (reynders) [reply frontend/e2e-test/test/e2e/specs/0_citizen/citizen-decisions.spec.ts:29]: Yes - which is "await Fixture.cleanup()" which is called in the suite cleanup. But agree that it is not obvious, so will add a redundant call via the function returned by initializeAreaAndPersonData 👍

Comment (jaahon) [reply frontend/e2e-test/test/e2e/specs/0_citizen/citizen-decisions.spec.ts]: Oh, nevermind then. You don't need to add the function call, I just didn't know `Fixture.cleanup` does that. There's something else wrong with the clean up then. It definitely has worked for other test fixtures before.

**Review by jaahon — APPROVED** [review]

**Pull request merged**